server/src/main/scala/org/scassandra/server/priming/prepared/PrimePrepared.scala

```diff
@@ -109,7 +109,7 @@ case class ExactMatch(matcher: Option[Any]) extends VariableMatch with LazyLoggi
     (variable, matcher) match {
       case (Some(v), Some(x)) if dataType.native.isDefinedAt(x) =>
         dataType.native(x).equals(v) // convert into data type's native value and then compare.
-      case (Some(v), Some(x)) =>
+      case (Some(_), Some(_)) =>
```

Comment:
Isn't x still needed here to support the logging statement below?

Reply:
Yep, we'd lost the s macro hence it was compiling